controllers/nstemplateset/space_roles.go

```diff
@@ -61,33 +62,36 @@ func (r *spaceRolesManager) ensure(logger logr.Logger, nsTmplSet *toolchainv1alp
 			toolchainv1alpha1.ProviderLabelKey: toolchainv1alpha1.ProviderLabelValue,
 			toolchainv1alpha1.OwnerLabelKey:    nsTmplSet.GetName(),
 		}
-		logger.Info("creating space role objects")
+		logger.Info("creating space role objects", "count", len(spaceRoleObjs))
```

Comment:
This is a minor detail but I guess it would be more clear to say 'applying' since `len(spaceRoleObjs)` is the expected number of objects rather than the number we will actually create right? eg. If 1 object is deleted we would only expect to create 1

```suggestion
		logger.Info("applying space role objects", "count", len(spaceRoleObjs))
```

Reply:
good point!